docs/core_docs/docs/integrations/document_loaders/web_loaders/firecrawl.mdx

```diff
@@ -0,0 +1,45 @@
+---
+hide_table_of_contents: true
+---
+
+# Firecrawl
+
+This guide shows how to use [Firecrawl](https://firecrawl.dev) with LangChain to load web data into an LLM-ready format using Firecrawl.
+
+## Overview
+
+[FireCrawl](https://firecrawl.dev) crawls and convert any website into LLM-ready data. It crawls all accessible subpages and give you clean markdown and metadata for each. No sitemap required.
+
+FireCrawl handles complex tasks such as reverse proxies, caching, rate limits, and content blocked by JavaScript. Built by the [mendable.ai](https://mendable.ai) team.
+
+This guide shows how to scrap and crawl entire websites and load them using the `FireCrawlLoader` in LangChain.
+
+## Setup
+
+You'll need to sign up and retrieve your [FireCrawl API key](https://firecrawl.dev).
+
+
+## Usage
+
+Here's an example of how to use the `FireCrawlLoader` to load web search results:
+
+Firecrawl offers 2 modes: `scrape` and `crawl`. In `scrape` mode, Firecrawl will only scrape the page you provide. In `crawl` mode, Firecrawl will crawl the entire website.
+
+
+import CodeBlock from "@theme/CodeBlock";
+import Example from "@examples/document_loaders/firecrawl.ts";
+
+import IntegrationInstallTooltip from "@mdx_components/integration_install_tooltip.mdx";
+
+<IntegrationInstallTooltip></IntegrationInstallTooltip>
```

Comment:
No need for an `IntegrationInstallTooltip` for non LC deps. You only this for langchain package installs